pkg/controller/master/virtualmachine/vm_controller.go

```diff
@@ -94,8 +96,11 @@ func (h *VMController) createPVCsFromAnnotation(_ string, vm *kubevirtv1.Virtual
 		// create DataVolume for PVC (only for CDI backend)
 		if createPVCWithDataVolume {
 			if _, err := h.dataVolumeClient.Get(vm.Namespace, pvcAnno.Name, metav1.GetOptions{}); apierrors.IsNotFound(err) {
-				targetSCName := *pvcAnno.Spec.StorageClassName
-				if err := h.createDataVolume(pvcAnno.Name, vm.Namespace, imageName, imageNS, targetSCName, pvcAnno.Spec.Resources); err != nil {
+				targetSC, err := h.scCache.Get(*pvcAnno.Spec.StorageClassName)
```

Comment:
In theory, `StorageClassName` could be nil

```
type PersistentVolumeClaimSpec struct {
...
	// storageClassName is the name of the StorageClass required by the claim.
	// More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1
	// +optional
	StorageClassName *string `json:"storageClassName,omitempty" protobuf:"bytes,5,opt,name=storageClassName"`
```

Reply:
thanks @w13915984028, 
I improved this. Now, if the StorageClassName is nil, I will try to find the default storageclass.